nuxeo-lambda-image-conversion/src/test/java/org/nuxeo/lambda/image/conversion/TestLambdaPictureConversion.java

```diff
@@ -21,48 +21,30 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
-
-import java.io.File;
-import java.io.Serializable;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.Executors;
-
-import javax.inject.Inject;
-
-import org.codehaus.jettison.json.JSONArray;
-import org.codehaus.jettison.json.JSONObject;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.nuxeo.ecm.core.api.Blob;
 import org.nuxeo.ecm.core.api.CoreSession;
 import org.nuxeo.ecm.core.api.DocumentModel;
 import org.nuxeo.ecm.core.api.impl.blob.FileBlob;
-import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
-import org.nuxeo.ecm.core.blob.BlobManager;
-import org.nuxeo.ecm.core.blob.BlobProvider;
-import org.nuxeo.ecm.core.blob.binary.Binary;
-import org.nuxeo.ecm.core.blob.binary.BinaryManager;
 import org.nuxeo.ecm.core.test.CoreFeature;
-import org.nuxeo.ecm.platform.test.PlatformFeature;
-import org.nuxeo.lambda.core.AbstractLambdaCaller;
-import org.nuxeo.lambda.core.LambdaService;
-import org.nuxeo.lambda.core.LambdaService.LambdaStatus;
-import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.lambda.image.conversion.common.LambdaFeature;
+import org.nuxeo.lambda.image.conversion.common.MemLambdaCaller;
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
-import org.nuxeo.runtime.test.runner.LocalDeploy;
 import org.nuxeo.runtime.transaction.TransactionHelper;
 
+import javax.inject.Inject;
+import java.io.File;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
 @RunWith(FeaturesRunner.class)
-@Features({ PlatformFeature.class })
-@Deploy({ "org.nuxeo.lambda.core", "org.nuxeo.lambda.image.conversion", "org.nuxeo.ecm.platform.picture.api",
-                "org.nuxeo.ecm.platform.picture.core", "org.nuxeo.ecm.platform.tag", "org.nuxeo.ecm.core.cache",
-                "org.nuxeo.ecm.automation.core" })
-@LocalDeploy("org.nuxeo.lambda.image.conversion.test:OSGI-INF/lambda-picture-config-override-contrib.xml")
+@Features({ LambdaFeature.class })
```

Comment:
As there's only one feature you can write it without braces: `@Features(LambdaFeature.class)` (several places)